mobile_config/src/main.rs

```diff
@@ -112,11 +106,7 @@ impl Daemon {
             .serve_with_shutdown(listen_addr, shutdown_listener)
             .map_err(Error::from);
 
-        tokio::try_join!(
-            pool_handle.map_err(Error::from),
-            md_pool_handle.map_err(Error::from),
-            server,
-        )?;
+        tokio::try_join!(server)?;
```

Comment:
same deal as iot_config; you can just await the server builder task instead of the try_join

Reply:
i see you fixed, ty